### DISCUSSION TITLE
Word CASES in the output of large-v3

### DISCUSSION THREAD

**Discussion by Kirankumar2609** [openai/whisper#1859] [General]
The output from large-v3 model does not have the correct CASE for obvious words. Here are some examples:

![image](https://github.com/openai/whisper/assets/85477926/9339b2b0-fb14-4f0f-958b-2947195d6afc)
![image](https://github.com/openai/whisper/assets/85477926/0b735825-a835-4ec1-a6c7-39c8a492269f)

Pronouns like "I" are not in Capital CASE in the latest model's output. 

1. Is anyone experiencing the same?
2. Which model works fine for translation of Indian languages? large-v2 (or) large-v3

**Comment by Chiranjeevraja** [comment]
+1

**Comment by glangford** [comment]
Are you using `--initial_prompt`?

* https://github.com/openai/whisper/discussions/625

**Reply by Kirankumar2609** [reply]
No

**Reply by lumpidu** [reply]
This is commonly known as Inverse text normalization (ITN). This is not done by the whisper model itself but is in general  implemented into an own model that is closely executed after the inital transcription and is of course totally language- and even domain dependent.

**Reply by ryanheise** [reply]
> This is not done by the whisper model itself

The Whisper model is capable of lowercasing and omitting punctuation on its own, and has tendencies that can be influenced by the initial prompt. As far as I can see, there is no code in Whisper that takes the output of the whisper model and converts it to lowercase in post processing (and this would be unnecessary to explain the phenomenon since the model itself can exhibit this behaviour).
